Airports/EGLL/RMA.txt

```diff
@@ -9,6 +9,8 @@ Heathrow RMA Easterlies     S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00
                             N051.43.34.000 W000.32.59.000 N051.38.46.000 W000.27.41.000
                             N051.38.46.000 W000.27.41.000 N051.38.46.000 E000.09.06.000
                             N051.38.46.000 W000.02.28.971 N051.36.34.932 E000.01.09.449
+                            N051.28.17.049 W000.27.42.990 N051.38.47.386 W000.27.06.403
```

Comment:
Those lines on the RMA aren't strictly correct, nor are they marked on the real world display. I'd prefer without :/

(The top one should actually be a line on radial 005 of the LON DME, and the bottom one radial 210 - you can see LON is marked on your display, but isn't drawn from?)

Reply:
Ok I can remove them, was just a thought to help depict the RMA.